sound/hybrid_font.h

```diff
@@ -181,6 +185,17 @@ class HybridFont : public SaberBase {
 	  drag.files_found()) {
 	e = &drag;
       }
+	    
+      // ultra - dynalock lockon
+      if (lockon.files_found()) {
+        PlayPolyphonic(&lockon);
+      } else {
+        if (config_.clashLockOn) { // ultra
+          PlayPolyphonic(&clsh); // clash fallback
+        }
+      }
+        // ultra
```

Comment:
please remove

Reply:
um no im not doing the coding im just sharing what i did? but its cool if you wanna change it!!